netlify-plugin-cloudinary/tests/lib/cloudinary-cname.test.js

```diff
@@ -49,7 +49,8 @@ describe('lib/util', () => {
       const { html } = await updateHtmlImagesToCloudinary(sourceHtml, {
         deliveryType: 'fetch',
         localDir: 'tests',
-        remoteHost: 'https://cloudinary.netlify.app'
+        remoteHost: 'https://cloudinary.netlify.app',
+        loadingStrategy: 'lazy'
```

Comment:
would these technically be needed in the tests if it's a default?

i wonder if alternatively, it makes sense to create a duplicate test where you change this to `eager`

Reply:
oops this is correct being here because it's an input that gets passed into `update*`, so nevermind this! :)